### PR TITLE
Create global store for walledId and added them to the necessary page

### PR DESCRIPTION
**Pull request by Luluameh**
Changes Made  are 

Global Store Creation

Added a new global store using Zustand (useWalletStore) to manage walletId and setWalletId actions.
Ensured the store initializes walletId from localStorage and synchronizes updates to localStorage.
Code Refactoring

Replaced prop drilling of walletId with direct access to the global store.
Update all affected components and hooks, including useDashboardData, to utilize the global store.

### CODE REVIEW

File: frontend/src/App.jsx [hunk 1]
```diff
@@ -13,9 +13,9 @@ import { connectWallet, logout, checkForCRMToken } from 'services/wallet';
 import { saveTelegramUser, getTelegramUserWalletId } from 'services/telegram';
 import Documentation from 'pages/spotnet/documentation/Documentation';
 import Withdraw from 'pages/vault/withdraw/Withdraw';
-
+import useWalletStore from 'stores /useWalletStore';
```
Comment (whateverfw) [inline frontend/src/App.jsx:16]: fix imports here and everywhere, seems like `useWalletStore` file doesn't exist?

**Review by whateverfw — CHANGES_REQUESTED** [review]
please fix failed build, also `useWalletStore` is missing

**Review by whateverfw — CHANGES_REQUESTED** [review]

**Comment by Luluameh** [comment]
Noted

**Review by whateverfw — APPROVED** [review]
@Luluameh Approved. Great job, thanks for quick implementation and comments resolvments.

**Comment by djeck1432** [comment]
@Luluameh Please, resolve merge conflict and I will merge it

**Comment by whateverfw** [comment]
@Luluameh seems like there are more conflicts

File: frontend/src/App.jsx [hunk 2]
```diff
@@ -13,28 +13,23 @@ import { logout } from 'services/wallet';
 import { saveTelegramUser, getTelegramUserWalletId } from 'services/telegram';
 import Documentation from 'pages/spotnet/documentation/Documentation';
 import Withdraw from 'pages/vault/withdraw/Withdraw';
-import { useConnectWallet } from 'hooks/useConnectWallet';
+import { useWalletStore } from 'stores/useWalletStore'; 
 import { Notifier } from 'components/Notifier/Notifier';
+import { useConnectWallet } from 'hooks/useConnectWallet ';
```
Comment (whateverfw) [inline frontend/src/App.jsx:18]: remove space in the import, it causing build errors

**Review by whateverfw — CHANGES_REQUESTED** [review]

**Review by whateverfw — APPROVED** [review]

**Review by djeck1432 — APPROVED** [review]
Looks good to me,thank you for quick job

**Pull request merged**